### ISSUE TITLE
Unexpected endpoint text behavior change when updating from @scalar/api-reference:1.0.2

### ISSUE THREAD

**Issue by csm-kb** [scalar/scalar#506]
### Expected Behavior

With 1.0.2, we were seeing our endpoints as HTTP paths (`/path/to-endpoint`) in the content sidebar and as headers in the main view.

### Actual Behavior

When we updated to the latest version, the `ApiReference` Vue component started using the `operationId` instead (such as `path_ToEndpoint`) -- this shift was unexpected (and not desired, in our case!).

I saw that this is the new behavior in recent changes in [parse.ts](https://github.com/scalar/scalar/blob/main/packages/swagger-parser/src/helpers/parse.ts) (order of use being summary -> operation ID -> path now), but
- it would be awesome if we could specify which of those we'd like to use for the endpoint reference text in the `ApiReference` config object! Our end-users prefer the original path 😁 

### Reproduction steps

```bash
1. Use a simple Vue app with the `ApiReference` component from `@scalar/api-reference:1.0.2`.
2. Pass the component this OpenAPI spec:

openapi: 3.0.1
info:
  title: Sample API
  version: 'dev'
paths:
  /path/to-endpoint:
    get:
      description: |
        A sample endpoint.
      operationId: path_ToEndpoint
      tags:
        - SampleTag
      responses:
        '200':
          description: ''
          content:
            application/json:
              schema:
                $ref: '#/components/schemas/SampleResponseBody'
              examples:
                Example1:
                  value:
                    id: 0x10c
components:
  schemas:
    SampleResponseBody:
      title: SampleResponseBody
      type: object
      description: Sample response body.
      properties:
        id:
          type: string
      required:
        - id
servers:
  - url: https://api.example.com
    description: Production
```
3. Observe that the reference content sidebar renders the endpoint with its path (`/path/to-endpoint`).
4. Update `@scalar/api-reference` to the latest version.
5. Observe that the reference content sidebar renders the endpoint with its operation ID instead (`path_ToEndpoint`).
```


### Version

1.1.3

### What browsers are you seeing the problem on?

_No response_

**Comment by hanspagel** [comment]
Thanks for reporting! Yes, that was a mistake in #448. We just merged #500, which makes the summary the default. But I see your point. Let me think about this. We’ll add another update soon.

**Comment by hanspagel** [comment]
Let’s try this: https://github.com/scalar/scalar/pull/507
